workspace/dnd.py

```diff
@@ -6,6 +8,7 @@
 from city import City
 
 parser = argparse.ArgumentParser(prog='dnd')
+introducing_aBadlyMadeVariableName = "Yes"
 
 # Constants
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename function "introducingABadlyMadeVariableName" to match the regular expression ^[a-z_][a-z0-9_]{2,30}$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](binarytiger.me/coding_rules#rule_key=python%3AS1542)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a docstring to this function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](binarytiger.me/coding_rules#rule_key=python%3AS1720)